packages/medusa/src/models/discount-condition.ts

```diff
@@ -126,6 +128,9 @@ export class DiscountCondition extends SoftDeletableEntity {
   })
   customer_groups: CustomerGroup[]
 
+  @OneToMany(() => Discount, (discount) => discount.rule_id)
+  discounts: Discount[]
```

Comment:
@adrien2p wuuut? Why was this never there?

Reply:
no idea 😂

Reply:
thought: Why is this added? A discount condition does not belong to a Discount but a DiscountRule.

Also, this addition defines a relationship between Discount and DiscountRule and is not related to the conditions, so I am not sure what the intention is here.

Reply:
We already have the relationship with DiscountRule